Makefile

```diff
@@ -532,6 +532,9 @@ docker-compose-sources: .git/hooks/pre-commit
             $(EXTRA_SOURCES_ANSIBLE_OPTS)
 
 docker-compose: awx/projects docker-compose-sources
+	ansible-galaxy install -r tools/docker-compose/ansible/requirements.yml;
+	ansible-playbook -i tools/docker-compose/inventory tools/docker-compose/ansible/initialize_containers.yml \
```

Comment:
The dev-env check can't find the `initialize_containers.yml` playbook, and I can't either.

Reply:
oh, ha, it's `initalize_containers.yml` elsewhere (sp)

Reply:
Fixed.